src/test/moves/retaliate.test.ts

```diff
@@ -0,0 +1,52 @@
+import { afterEach, beforeAll, beforeEach, describe, expect, it } from "vitest";
+import Phaser from "phaser";
+import GameManager from "#app/test/utils/gameManager";
+import { Species } from "#enums/species";
+import { Moves } from "#enums/moves";
+import { getMovePosition } from "#app/test/utils/gameManagerUtils";
+import { allMoves } from "#app/data/move.js";
+
+describe("Moves - Retaliate", () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  const retaliate = allMoves[Moves.RETALIATE];
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+  });
+
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+    game.override.battleType("single");
+    game.override.enemySpecies(Species.SNORLAX);
+    game.override.enemyMoveset([Moves.RETALIATE, Moves.RETALIATE, Moves.RETALIATE, Moves.RETALIATE]);
+    game.override.enemyLevel(100);
+
+    game.override.moveset([Moves.RETALIATE, Moves.SPLASH]);
+    game.override.startingHeldItems([{name: "WIDE_LENS", count: 3}]);
+    game.override.startingLevel(80);
+    game.override.disableCrits();
```

Comment:
```suggestion
    game.override
      .battleType("single")
      .enemySpecies(Species.SNORLAX)
      .enemyMoveset([Moves.RETALIATE, Moves.RETALIATE, Moves.RETALIATE, Moves.RETALIATE])
      .enemyLevel(100)
      .moveset([Moves.RETALIATE, Moves.SPLASH])
      .startingHeldItems([{name: "WIDE_LENS", count: 3}])
      .startingLevel(80)
      .disableCrits();
```

You can use chaining here

Reply:
ty